### PR TITLE
added 6-get-article-comments

### PR DESCRIPTION
**Pull request by nickdip**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -115,4 +115,74 @@ describe("GET /api/articles", () => {
...
+            expect(msg).toBe("Article not found")
+        })})
+    test("400: invalid article_id", () => {
+        return request(app)
+        .get("/api/articles/northcoders/comments")
+        .expect(400)
+        .then( ( { body: { msg } } ) => {
+            expect(msg).toBe("Invalid article_id")
+        })})
+})
```
Comment (yalicee) [inline __tests__/app.test.js:188]: Suggestion: To improve testing you should add a test for an empty array of comments

**Review by yalicee — APPROVED** [review]
Great code as per usual just check my comment before you merge
